recipes/tsflex/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "tsflex" %}
+{% set version = "0.1.1.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tsflex-{{ version }}.tar.gz
+  sha256: 1215b19caa19749a26e4c77420a415127b037ff600907435336c0180fbe8dd8f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python >=3.6,<4.0
+  run:
+    - dill >=0.3.3,<0.4.0
+    - fastparquet >=0.6.3,<0.7.0
+    - numpy >=1.19.0,<2.0.0
+    - pandas >=1.2.3,<2.0.0
+    - pathos >=0.2.7,<0.3.0
+    - python >=3.6,<4.0
+    - tqdm >=4.60.0,<5.0.0
+
+test:
+  imports:
+    - tsflex
+    - tsflex.chunking
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/predict-idlab/tsflex
+  summary: Toolkit for flexible processing & feature extraction on time-series data
+  license: MIT
+  license_file: LICENSE
+  doc_url: https://predict-idlab.github.io/tsflex/
+  dev_url: https://github.com/predict-idlab/tsflex
+
+extra:
+  recipe-maintainers:
+    - jvdd
+    - jonasvdd
```

Comment:
We need all maintainers to agree to maintain the recipe. @jonasvdd are you ok being a maintainer here?

Reply:
Yes I am! :)

Reply:
@conda-forge-admin, please ping team